sources/packages/web/src/types/contracts/DesignationAgreementContract.ts

```diff
@@ -12,15 +12,22 @@ export interface GetDesignationAgreementDto {
   designationId: number;
   designationStatus: DesignationAgreementStatus;
   locationsDesignations: LocationsDesignationsDto[];
+  startDate: Date;
+  endDate: Date;
   submittedData: any;
```

Comment:
why `any`?looks like not part of this PR

Reply:
It is completely dynamic and there is no processing at the API level. The value here is directly saved to DB and retrieved from DB with the only purpose of populating the same dynamic area in the form.

Reply:
oh, it's `submittedData`, my bad I read it as `submittedDatE`. resolving the comment 😜